src/components/checkbox/index.tsx

```diff
@@ -155,33 +172,38 @@ class Checkbox extends Component<CheckboxPropTypes, CheckboxState> {
   }
 
   render() {
-    const {selectedIcon, color, iconColor, disabled, testID, style, ...others} = this.props;
+    const {selectedIcon, color, iconColor, disabled, testID, label, labelStyle, style, ...others} = this.props;
     return (
-      // @ts-ignore
-      <TouchableOpacity
-        {...this.getAccessibilityProps()}
-        activeOpacity={1}
-        testID={testID}
-        {...others}
-        style={[this.getBorderStyle(), style]}
-        onPress={this.onPress}
-      >
-        {
-          <Animated.View
-            style={[this.styles.container, {backgroundColor: this.getColor()}, {opacity: this.animationStyle.opacity}]}
-          >
-            <Animated.Image
+      <TouchableOpacity row onPress={this.onPress}>
+        <Animated.View
+          {...this.getAccessibilityProps()}
```

Comment:
Some of these props belong to the most top container. 
For instance `activeOpacity` is relevant only for TouchableOpacity. 
Notice that now the behavior is a little different - pressing the checkbox comes with the default activeOpacity (0.6) and it doesn't look as before)